packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -519,7 +519,7 @@
     text-decoration: none;
 
     &:after {
-      background: var(--color-gray-light);
+      background: var(--text-input__border-color);
```

Comment:
I went down this path when troubleshooting and concluded that this is not the correct color. Because of the subtle blue backgroundn of the button, I think our designers opted for a lighter color. The image update must have happened on account of the gray shift that happened in an earlier PR (but for some reason the image didn't get flagged as changed). I think we should put this in its own component variable and assign it to the gray-light